packages/utils/src/internals/robots.ts

```diff
@@ -0,0 +1,58 @@
+import { downloadListOfUrls, gotScraping } from '@crawlee/utils';
```

Comment:
You'll need to import these from relative files, the module cannot import itself